src/Utility/Window.hpp

```diff
@@ -17,9 +18,15 @@ class Environment : public QObject {
   Q_PROPERTY(bool lockedCursor READ lockedCursor WRITE setLockedCursor NOTIFY
                  lockedCursorChanged)
   Q_PROPERTY(QString gitVersion READ gitVersion CONSTANT)
+  Q_PROPERTY(bool minimapOnLeft MEMBER m_minimapOnLeft NOTIFY minimapOnLeftChanged)
+  Q_PROPERTY(QColor playerIndicatorColor MEMBER m_playerIndicatorColor NOTIFY playerIndicatorColorChanged)
+  Q_PROPERTY(QPointF playerLocation READ playerLocation NOTIFY playerLocationChanged)
```

Comment:
generalnie to te wszystkie nowe wlasciwosci to bardziej naleza do World niz do calego srodowiska(Environment)